openml/tasks/functions.py

```diff
@@ -65,9 +63,18 @@ def get_estimation_procedure_list():
         "estimationprocedure/list")
     procs_dict = xmltodict.parse(xml_string)
     # Minimalistic check if the XML is useful
-    assert procs_dict['oml:estimationprocedures']['@xmlns:oml'] == \
-        'http://openml.org/openml'
-    assert type(procs_dict['oml:estimationprocedures']['oml:estimationprocedure']) == list
+    if 'oml:estimationprocedures' not in procs_dict:
```

Comment:
I guess there was some debugging here? This seems pretty heavy handed but it it was useful at some point, maybe it's necessary. I wonder if this can be generalized.